tt_eager/tt_dnn/op_library/eltwise_unary/eltwise_unary_op.hpp

```diff
@@ -373,6 +375,7 @@ constexpr auto elu = make_eltwise_unary_with_param<UnaryOpType::ELU>{};
 constexpr auto heaviside = make_eltwise_unary_with_param<UnaryOpType::HEAVISIDE>{};
 constexpr auto right_shift = make_eltwise_unary_with_param<UnaryOpType::RIGHT_SHIFT>{};
 constexpr auto left_shift = make_eltwise_unary_with_param<UnaryOpType::LEFT_SHIFT>{};
+constexpr auto remainder = make_eltwise_unary_with_param<UnaryOpType::REMAINDER>{};
```

Comment:
I think we might want to consider adding an assertion in validate for these ops that are only supported on a specific arch.
Ex something like this.

```
inline void validate_supported_arch(tt::ARCH arch, UnaryOpType op_type) {
    switch (op_type) {
        case UnaryOpType::REMAINDER:
        case UnaryOpType::FLOOR:
            TT_FATAL(arch == tt::ARCH::WORMHOLE_B0, "Op is only supported on Wormhole");
            break;
        default:
            return;
    }
}
```

and in validate
```
auto arch = input_tensor_a.device()->arch();
for (const auto& unary_op : this->op_chain) {
    validate_supported_arch(arch, unary_op.op_type);
}
```

Reply:
agree with this, we have a lot of new ops that are wormhole b0 only. @Aswinmcw @VirdhatchaniKN adding the above asserts would be safer than just docs. And can you add them for floor, trunc, shift and all those other ops too?

Reply:
Sure, we will add these asserts in the ops